docs/data/sdks/ios-swift/ampli.md

```diff
@@ -0,0 +1,206 @@
+---
+title: iOS Ampli Wrapper
+description: Learn how to install and use the Ampli Wrapper for the iOS Swift runtime. 
+---
+
+--8<-- "includes/ampli/ampli-overview-section.md"
+
+Amplitude Data supports tracking analytics events from iOS apps written in Swift.
+
+!!!warning "Objective-C Not Supported"
+    Please note that the [latest iOS SDK](/data/sdks/ios-swift) is NOT compatible with Objective-C projects. Use the [maintenance iOS SDK](/data/sdks/ios) with [maintenance Ampli](/data/sdks/ios/ampli) if your project requires compatibility with Objective-C.
```

Comment:
The newest version of Ampli is built on the latest iOS SDK, which lacks compatibility with ...